sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
@nickelsen don't let logic get in the way of empirical examination.  In particular, don't assume that Blizzard changed nothing in the meantime.

https://www.reddit.com/r/starcraft/comments/3wzhkg/310_patch_fixes_apm_sc2_now_shows_your_real_apm/

I believe we can separate APM calculation and visualization into two independent tasks.  However you might find it easier to just fix APM match what Bliz says in-game than to try and keep the tasks separate.